src/components/Filter/FilterInput.vue

```diff
@@ -76,7 +76,7 @@
               @keydown.left="leftKeyInputHandler"
               @keydown.right="rightKeyInputHandler"
               @keydown.delete="deleteHandler"
-              @keydown.meta.a.prevent="selectInputAndTags"
+              @keydown.meta.a.prevent.stop="selectInputAndTags"
```

Comment:
Why are you stopping the event propagation?

Reply:
The event in the QN filter input is bubbling up and selecting the entire DOM in `cmd+a` instead of just the input content, that's why I'm stopping the event propagation.

Reply:
Why doesnt it do that in other places we use the input though 🤔

Reply:
Because `GenericModal` overwrites the behavior of cmd+a to select the content of the modal